### PR TITLE
Clarify AWS installation flow

### PR DESCRIPTION
**Pull request by keirthana**
To provide an end to end flow for the appliance installation workflow on AWS.

### CODE REVIEW

**Comment by keirthana** [comment]
@carlosbravoa Please have a look and let me know your feedback

File: howto/install-appliance/install-on-aws.md [hunk 1]
```diff
@@ -140,12 +140,31 @@ When the instance is successfully launched, you can find its public IP address i
...
+## Install additional packages
 
-Perform the following steps to finish the appliance installation on the virtual machine. If you are not already familiar with how to perform these steps, see {ref}`tut-installing-appliance` for detailed instructions.
+Some preparation is required for using Anbox Cloud that involves installing additional packages. To prepare your machine for further steps, run:
 
-1. Prepare the virtual machine and install additional required packages. See {ref}`sec-prepare-machine` for instructions.
-1. Initialise the appliance. See {ref}`sec-initialise-appliance` for instructions.
-1. Register your Ubuntu SSO account with the appliance dashboard. See {ref}`sec-register-dashboard` for instructions.
+    anbox-cloud-appliance prepare-node-script
+    anbox-cloud-appliance prepare-node-script | sudo bash -ex
```
Comment (carlosbravoa) [inline howto/install-appliance/install-on-aws.md:148]: If I understand correctly, these two instructions are "either-or". We may need to add a few lines to explain which one to use.

**Review by carlosbravoa — COMMENTED** [review]
There is only one comment when running `anbox-cloud-appliance prepare-node-script`

File: howto/install-appliance/install-on-aws.md (hunk 1 above)
Comment (keirthana) [reply howto/install-appliance/install-on-aws.md:147]: You are right, the first one is not needed unless you want to look at the script. That can be for the deeper learning experience detailed in the linked tutorial. For a how-to guide that intends to run through the exact steps of deployment, applying the script is enough.

File: howto/install-appliance/install-on-aws.md [hunk 2]
```diff
@@ -140,12 +140,30 @@ When the instance is successfully launched, you can find its public IP address i
 
 Connect to the virtual machine hosting the appliance using SSH. To do so, use the user name `ubuntu` and provide the path to your private key file. See [Connect to your Linux instance using SSH](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html) for instructions on how to connect.
 
-## Finish the installation
+## Install additional packages
```
Comment (morphis) [inline howto/install-appliance/install-on-aws.md:143]: @keirthana we talked about extending the init command to point out a missing `pro enable`. However that will take until 1.24.1/2. Should we just document it in the meantime here?

**Review by morphis — CHANGES_REQUESTED** [review]

File: howto/install-appliance/install-on-aws.md [hunk 3]
```diff
@@ -140,12 +140,30 @@ When the instance is successfully launched, you can find its public IP address i
...
+
+To initialise the appliance, run:
+
+    sudo anbox-cloud-appliance init
+
+You will be asked a few questions when you run the `init` command. Accept the default answers if you do not want to make any changes.
+
+## Register with the dashboard
+
+After initialisation, you must register your user account with the Anbox Cloud dashboard to access it. Run:
```
[the remark this answers is not in the document]
Comment (keirthana) [reply howto/install-appliance/install-on-aws.md:161]: We have it as part of the requirements even before someone starts the procedure. (https://github.com/canonical/anbox-cloud-docs/pull/226/files#diff-8327e7ac3fd8363d2e299b5dcdcf0d6153002e096737c7ac6e784ceeba9c7040R50)

File: howto/install-appliance/install-on-aws.md (hunk 2 above)
Comment (keirthana) [reply howto/install-appliance/install-on-aws.md:155]: Yes, did that now. Thanks!

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**